src/char/int_guild.hpp

```diff
@@ -8,6 +8,7 @@
 
 #include <common/cbasetypes.hpp>
 #include <common/database.hpp>
+#include "common/mmo.hpp"
```

Comment:
This is actually the preferred way of importing project headers, so there will be another pr to handle them all.